### DISCUSSION TITLE
JUnit5

### DISCUSSION THREAD

**Discussion by phillipross** [arquillian/arquillian-core#403] [Q&A] [answered]
How's JUnit5 support for arquillian coming along?

**Comment by bartoszmajsak** [comment, marked as answer]
There's an initial cut in Alpha releases, but otherwise, I'm not aware of any active development right now. 

@starksm64 @chengfang do you know more?

**Reply by chengfang** [reply]
arquillian-junit5 Alpha releases have been used by various project, such as https://github.com/eclipse-ee4j/batch-tck. It seems to be working well. Are there any specific features or bugs in your mind, @phillipross ?

**Reply by phillipross** [reply]
Nothing specifically, mostly curious as to wether or not it was in a usable state.  But thanks for the batch-tck reference, and am interested in knowing what other opensource projects might be relying upon it.  Thanks again!
